packages/types/src/sign-client/engine.ts

```diff
@@ -214,6 +214,19 @@ export interface EnginePrivate {
 
   setProposal(id: number, proposal: ProposalTypes.Struct): Promise<void>;
 
+  setAuthRequest(
+    id: number,
+    params: {
+      request: {
```

Comment:
Seems like this is basically a straight extension of the `SessionAuthenticateRequestParams` interface (https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/packages/types/src/sign-client/auth.ts#L158) with `verifyContext` added in.

Might benefit from declaring a new interface e.g. `SessionAuthenticateRequest extends SessionAuthenticateRequestParams` so these two types are linked properly.

Otherwise if we ever change `SessionAuthenticateRequestParams ` this inline typing needs to be updated separately.

Reply:
good call 👍